examples/tests/data_silo_plugin/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    transcend = {
+      version = "0.5.1"
+      source  = "transcend.com/cli/transcend"
+    }
+  }
+}
+
+provider "transcend" {
+  url = "https://api.dev.trancsend.com/"
+}
+
+variable "schedule_frequency" {
```

Comment:
Maybe name this something like `schedule_frequency_minutes` or something with the unit included in the name